From 96c86bf6e73ae87472cb9ae39a1c5e019b722795 Mon Sep 17 00:00:00 2001
From: Alon Levy <alevy@redhat.com>
Date: Fri, 4 Feb 2011 12:15:01 -0200
Subject: [PATCH 06/15] qdev: add data pointer to Property

RH-Author: Alon Levy <alevy@redhat.com>
Message-id: <1296821710-5316-3-git-send-email-alevy@redhat.com>
Patchwork-id: 17725
O-Subject: [PATCHv4 RHEL6.1 qemu-kvm 02/11] qdev: add data pointer to Property
Bugzilla: 641833
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>

For later use by PROP_TYPE_ENUM, will store enumeration name/value
table there.

Signed-off-by: Alon Levy <alevy@redhat.com>
---

upstream status: http://patchwork.ozlabs.org/patch/81736/
---
 hw/qdev.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>
---
 hw/qdev.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/hw/qdev.h b/hw/qdev.h
index d6ce3de..353e38a 100644
--- a/hw/qdev.h
+++ b/hw/qdev.h
@@ -80,6 +80,7 @@ struct Property {
     int          offset;
     int          bitnr;
     void         *defval;
+    void         *data;
 };
 
 enum PropertyType {
-- 
1.7.4.rc1.16.gd2f15e

